src/basic/string-table.h

```diff
@@ -30,45 +29,44 @@ ssize_t string_table_lookup_from_string_fallback(const char * const *table, size
                 return (type) string_table_lookup_from_string_with_boolean(name##_table, ELEMENTSOF(name##_table), s, yes); \
         }
 
-#define _DEFINE_STRING_TABLE_LOOKUP_TO_STRING_FALLBACK(name, type, max, scope)                          \
-        /* NOLINTNEXTLINE (readability-inconsistent-declaration-parameter-name) */                      \
-        scope int name##_to_string_alloc(type i, char **ret) {                                          \
-                return string_table_lookup_to_string_fallback(name##_table, ELEMENTSOF(name##_table), i, max, ret); \
+#define _DEFINE_STRING_TABLE_LOOKUP_TO_STRING_FALLBACK(name, type, arg, max, scope)                     \
+        scope int name##_to_string_alloc(type arg, char **ret) {                                        \
+                return string_table_lookup_to_string_fallback(name##_table, ELEMENTSOF(name##_table), arg, max, ret); \
         }
 
 #define _DEFINE_STRING_TABLE_LOOKUP_FROM_STRING_FALLBACK(name, type, max, scope) \
         scope type name##_from_string(const char *s) {                           \
                 return (type) string_table_lookup_from_string_fallback(name##_table, ELEMENTSOF(name##_table), s, max); \
         }
 
-#define _DEFINE_STRING_TABLE_LOOKUP(name, type, scope)                    \
-        _DEFINE_STRING_TABLE_LOOKUP_TO_STRING(name, type, scope)          \
+#define _DEFINE_STRING_TABLE_LOOKUP(name, type, arg, scope)                    \
```

Comment:
Frankly, I'd much more prefer if we just declare the prototypes in a macro too, and name the params consistently s/i. That also gives you one extra benefit that pure and const attributes can be implied everywhere